### PR TITLE
writePlistToString is deprecated use dumps

### PR DESCRIPTION
**Pull request by typemytype**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 fonttools==3.28.0
-ufoLib==2.2.2
+ufoLib==2.2.5
```
Comment (anthrotype) [inline requirements.txt]: but I haven't tagged it yet...

Comment (anthrotype) [reply requirements.txt]: please wait until tomorrow, thanks

Comment (typemytype) [reply requirements.txt]: oh, thats it, I wait until your done :)

thanks

**Comment by benkiel** [comment]
@typemytype I think this may be ready now.

**Pull request merged**